code/modules/gear_presets/uscm_ship.dm

```diff
@@ -33,7 +33,6 @@
 		ACCESS_WY_CORPORATE,
 		ACCESS_ILLEGAL_PIRATE,
 		ACCESS_MARINE_COMMAND,
-		ACCESS_MARINE_DROPSHIP,
```

Comment:
CL can still fly the shuttle due to ACCESS_WY_CORPORATE_DS, however _currently_ this access is insufficient to open the cockpit door.

Reply:
I removed the console access on the dropship consoles. That access is just not used for now but I may use it again later.